api/openapi.yaml

```diff
@@ -1759,6 +1768,18 @@ components:
       properties:
         state:
           type: string
+          description: |
+            Describes the state of TargetScan/ScanResult.
+
+            | State       | Description                                                                                       |
+            | ----------- | ------------------------------------------------------------------------------------------------- |
+            | NotScanned  | Scanners are skipped for ScanResult                                                               |
```

Comment:
When we refactor the ScanResult state we should have a different enum for an AssetScan's state and for the individual families states e.g. :
ScanResult = Pending, Scheduled, ReadyToScan, Aborted, Done, Failed
A family's result state = NotScanned, Pending, InProgress, Done, Failed

Reply:
Agreed! Will close this PR and add the docs to the one which will include the refactor.